packages/mask/background/services/helper/popup-opener.ts

```diff
@@ -49,7 +49,7 @@ async function openWindow(url: string): Promise<void> {
         const { id } = await browser.windows.create({
             url: browser.runtime.getURL(url),
             width: 400,
-            height: 600,
+            height: 628,
```

Comment:
Height of popup can't be larger than `600`
https://developer.chrome.com/docs/extensions/reference/browserAction/#popup

Reply:
Yes, but the actual situation is that if the popup is set to 600px, the content page is a 572px height window.

Reply:
Yes, I know this. But the reality is that when you set it to 600, the height of the new window that opens will be 572p x (which I suspect is the browser's navigation height)